uint16 leftDist
uint16 rightDist
uint16 targetX
uint16 targetY
byte state1
byte state0
